apps/web/src/components/mapping/mapbox/map.tsx

```
@@ -1,18 +1,15 @@
 "use client";
 import { useMapStore } from "@/components/providers/map-state-provider";
 import { env } from "env";
-import { memo, useMemo } from "react";
-import { Map, MapProvider, Marker } from "react-map-gl/mapbox";
-import MarkerPin from "../google-maps/marker_pin";
 import "mapbox-gl/dist/mapbox-gl.css";
+import { memo, useMemo } from "react";
+import { Map, MapProvider } from "react-map-gl/mapbox";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)